annotations/views/rest_views.py

```diff
@@ -493,8 +500,76 @@ def _relabel(datum):
                 'identifier': 'uri'
             }
 
-            return {_fields.get(k, k): v for k, v in datum.iteritems()}
-        return Response({'results': map(_relabel, [c.data for c in concepts])})
+            return {_fields.get(k, k): v for k, v in datum.iteritems() }
+        results = map(_relabel, [c.data for c in concepts])
+        
+        for result in results:
+            if result["identities"]: # if identities exist append the first identitiy to the list so that we can filter out other identities against it
+                identities = result["identities"][0]["concepts"]
+            else:
+                identities = []
+            for ident in result["identities"]: # go through the identities in each result
+                for identity in identities: # go through the identities in the identities list
+                    if identity not in ident["concepts"]: # if the ideneities list does not contain the identity from the result then add it to the list
+                        identities.extend(ident["concepts"])
+            result["identities"] = identities # replace the identities list
+            if result["identities"]:
+                concepts = result["identities"]
+                uri = result["uri"]
+                if uri in concepts: concepts.remove(uri) # remove original uri from the list if it exists. 
+                i = 0 # used to generate concept name
+                new_concepts = []
+                for concept in concepts: # determine if the concept is a viaf or concept power uri
+                    #go through all the concepts and parse xml data for each concept
+                    #then append info to list and then append list to dictionary so that
+                    #list can be referenced as con0, con1, etc
+                    # This will have to be updated if a new concept Authority is added
+                    hps = re.search( r'www.digitalhps.org', concept, re.M|re.I)
+                    viaf = re.search( r'viaf.org', concept, re.M|re.I)
+                    if hps:
+                        url = concept
+                        data = urllib2.urlopen(url)
+                        tree = ET.parse(data)
+                        root = tree.getroot()
+                        namespace = {'hps': 'http://www.digitalhps.org/'}
```

Comment:
conceptpower supports json if you'd rather use that (just add an accept header in your request).